### PR TITLE
Refactor Markdown style of Readme files

### PR DESCRIPTION
**Pull request by santisoler**
Wrap lines to 80 characters per line when possible. Remove horizontal
rules from `README.md`. Make badge on translated README.md to return to
root instead to English `README.md`.

### CODE REVIEW

**Comment by cyb3rko** [comment]
Oh yeah, I already wondered how you could return to root, thanks for the improvements!

Why do you wrap the text? I've never seen this?

**Comment by santisoler** [comment]
> Why do you wrap the text? I've never seen this?

I wrap text to 80 characters per line, so I can open several files side by side. I do it mainly for code, but also for Markdown, HTML, LaTeX, etc. Although it's merely a matter of taste, some standards require you to wrap your code. For example, on Python you have the PEP8 Style Guide. We follow those guides for writing code so every code looks similar. Wrapping lines is one of the requirements.

Basically, it consists in adding line breaks before reaching the 80 chars. But, depending on the IDE or text editor you use, there could be a special tool for that. I use Neovim as my main text editor and IDE. Under VIM you have a keybinding to do so: `gq`.
If you are using another IDE, you should probably look for an quick way to do it.

**Comment by santisoler** [comment]
> Oh yeah, I already wondered how you could return to root, thanks for the improvements!

Yes, I found that solution on StackOverflow. But it's not optimal: it only returns to the root of the `master` branch, even if you're standing on a different one. It serves our purposes, though.

**Comment by cyb3rko** [comment]
> > Why do you wrap the text? I've never seen this?
> 
> I wrap text to 80 characters per line, so I can open several files side by side. I do it mainly for code, but also for Markdown, HTML, LaTeX, etc. Although it's merely a matter of taste, some standards require you to wrap your code. For example, on Python you have the PEP8 Style Guide. We follow those guides for writing code so every code looks similar. Wrapping lines is one of the requirements.
> 
> Basically, it consists in adding line breaks before reaching the 80 chars. But, depending on the IDE or text editor you use, there could be a special tool for that. I use Neovim as my main text editor and IDE. Under VIM you have a keybinding to do so: `gq`.
> If you are using another IDE, you should probably look for an quick way to do it.

Yeah, I already activated it. Thanks!
